code/modules/antagonists/revolution/revolution.dm

```diff
@@ -419,6 +419,9 @@
 			if (!(mind.assigned_role in GLOB.command_positions + GLOB.security_positions))
 				continue
 
+			if (!(mind.has_antag_datum(/datum/antagonist/rev)))
+				continue
```

Comment:
If i'm reading this right, this only adds "Enemy of the Revolution" if they ARE a rev, is this tested?
`(if mind does not have a rev antagonist datum, skip this player in the loop.)`